src/features/dsPreview/DatasetPreviewPage.tsx

```diff
@@ -57,11 +55,6 @@ const DatasetPreviewPage: React.FC<DatasetPreviewPageProps> = ({
     readmeContainerHeight = 'auto'
   }
 
-  useEffect(() => {
-    const ref = newQriRef({ username: qriRef.username, name: qriRef.name, path: qriRef.path })
-    dispatch(loadDsPreview(ref))
```

Comment:
mentioned in other comments, but we need to keep `loadDsPreview`. We can maybe come up with different metrics for when we go to fetch, though.

What are circumstances in which the "head" dataset would change?

1) if we are loading the page for the first time and no preview exists
2) if we are loading the page for that dataset for the first time (so different username/name combo. This may also get triggered if the dataset gets renamed, but I think that's fine)
3) if we have added a new version to the dataset. This can happen via the automation side when we commit a new version of the transform, if an automation runs, or if we do a manual update. In all three cases, we can potentially leverage the websocket events to know that a save occurred. If the save occurred, and we are currently viewing that dataset, we can trigger a `loadDsPreview`, to get the latest "head" dataset.

OR

we just go back to loading it every time we visit the page.